### DISCUSSION TITLE
CSVReader Error

### DISCUSSION THREAD

**Discussion by fervaldies** [idaholab/moose#27037] [Q&A General] [answered]
Hello, I am trying to run my code to read a CSV file to use it within an AuxKernels and also Materials block. I was using this code some time ago with the CSVReader and it was working correctly, but now it is giving the following error:

invalid float syntax for parameter: AuxKernels/add_material_in_the_ellipsoid/ry=Input_Params

My code is the following. First I am importing the CSV as:

[VectorPostprocessors]
  [./Input_Params]
    type = CSVReaderVectorPostprocessor
    csv_file = Input_Params_mm.csv
    #header = true
  [../]
[]

And then I am trying to use it in the AuxKernels as:

[AuxKernels]
  [./add_material_in_the_ellipsoid]
    type = FunctionPathEllipsoidAux
    variable = level_set_var
    vector_ry = ry # Column in Excel
    ry = Input_Params # Radius of ellipsoid in y (mm)
  [../]
[]

Is there now a new way to call the different columns and values?

Thank you in advance!

**Comment by GiudGiud** [comment]
Hello

Did you check the documentation page on the moose website for the functionEllipsoidAux?
That information, the new parameter names, would be there

**Reply by fervaldies** [reply]
I think the problem that I am having is that the CSV is not reading the data correctly, or that the method that I am using is not allowed anymore. The parameters names are correct I think, but I do not know how to select the correct column and value from the CSV file as before.

**Reply by GiudGiud** [reply, marked as answer]
There's a parameter to select that. 

A common issue with the CSV is if you opened it on a windows machine and the line endings got messed up

**Reply by fervaldies** [reply]
Oh solved! Thank you very much! :)
